packages/nimble-components/src/utilities/unit-format/decimal-unit-format.ts

```diff
@@ -33,10 +32,14 @@ export class DecimalUnitFormat extends UnitFormat<DecimalUnitFormatOptions> {
         }
     ) {
         super();
+        type SignDisplay = Intl.NumberFormatOptions['signDisplay'];
         const intlNumberFormatOptions = {
             maximumFractionDigits,
             minimumFractionDigits,
-            useGrouping: true
+            // Hack to avoid a ts error about signDisplay not accepting the value 'negative'.
+            // The value has been supported by browsers since 8/23, but TypeScript still hasn't
+            // added it to the type definitions. See https://github.com/microsoft/TypeScript/issues/56269
```

Comment:
Maybe it's just me, but I'm not a huge fan of calling something in our code a "hack". What do you think about calling this a "workaround" instead?

Same in the other similar comments.

Reply:
Ha! Okay, okay... it's a [workaround](https://en.wikipedia.org/wiki/A_rose_by_any_other_name_would_smell_as_sweet).